consensus/src/consensus.rs

```diff
@@ -157,7 +158,7 @@ impl<T: Operations + 'static, D: Database + 'static> Consensus<T, D> {
 
         tokio::spawn(async move {
             if ru.round > 0 {
-                future_msgs.lock().await.clear_round(ru.round - 1);
+                future_msgs.lock().await.remove_msgs_by_round(ru.round - 1);
```

Comment:
Should we remove messages belonging to any previous round?
 
Right now it should still work with current implementation, but if we remove the consensus spawn while syncing, this can lead to issues of not removing messages related to the skipped round

Reply:
This case has already been handled here https://github.com/dusk-network/rusk/blob/c14ad10795ab76288833d2cbb779b77a1d181169/node/src/chain/acceptor.rs#L573